morph/browser/extractMorphemes.py

```diff
@@ -17,7 +17,7 @@ def per( st, n ):
     if notecfg is None: return st
     morphemizer = getMorphemizerByName(notecfg['Morphemizer'])
     for f in notecfg['Fields']:
-        ms = getMorphemes(morphemizer, n[f], n.tags)
+        ms = getMorphemes(morphemizer, n[f], n.tags, cfg1('ignore grammar position'))
```

Comment:
How about you create a variable that stores the boolean `cfg1('ignore grammar position')` and then pass it into getMorphemes() as opposed to pulling from cfg1 individually everytime for every field in a note (same comment applies to other changes). That boolean is not going to change mid-process so it only needs to be retrieved once (not a groundbreaking improvement in speed for browser buttons, but perhaps it's noticeable on the main.py change since it's running for every note and recalcs can be for thousands of notes).

Reply:
Done.  I also made a change to the removePositions() implementation which, for some Pythonic reason I don't understand, made morpheme calculation run faster.